console/client/package.json

```diff
@@ -34,9 +34,11 @@
     "@monaco-editor/react": "4.5.2",
     "@tailwindcss/forms": "^0.5.6",
     "@vitejs/plugin-react": "^4.0.4",
+    "@viz-js/viz": "3.2.0",
     "highlight.js": "^11.8.0",
     "json-schema": "0.4.0",
     "json-schema-faker": "0.5.0-rcv.46",
+    "p5": "1.7.0",
```

Comment:
Hey @EdwardIrby! Are we still using `p5` for this PR? We can maybe review this if not.

Reply:
Nope good catch